frontend/src/components/Dashboard/SideBar.js

```diff
@@ -0,0 +1,261 @@
+import React from 'react';
+import cx from 'clsx';
+import { withStyles } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import Card from '@material-ui/core/Card';
+import Avatar from '@material-ui/core/Avatar';
+import Divider from '@material-ui/core/Divider';
+import LinearProgress from '@material-ui/core/LinearProgress';
+import $ from 'jquery';
+import { lighten } from '@material-ui/core/es/styles/colorManipulator';
+import { renderSize, renderDuration } from '../../util/worksheet_utils';
+
+const styles = ({ spacing, palette }) => {
+    const family =
+        '-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"';
```

Comment:
Does this font family look consistent with the rest of CodaLab (which is I think, by default, Roboto)?

Reply:
Updated